projects/core/src/lib/layers/symbology/symbology-creator/symbology-creator.component.ts

```diff
@@ -20,6 +20,7 @@ import {RasterSymbology} from '../symbology.model';
 export enum SymbologyCreationType {
     AS_INPUT = 'AS_INPUT',
     COMPUTE_LINEAR_GRADIENT = 'COMPUTE_LINEAR_GRADIENT',
+    LINEAR_GRADIENT_FROM_MIN_MAX = 'LINEAR_GRADIENT_FROM_MIN_MAX',
 }
 
```

Comment:
Could you add this? There are errors if you open up these operators that use the symbology creator.

```diff
diff --git a/projects/core/src/lib/operators/dialogs/interpolation/interpolation.component.html b/projects/core/src/lib/operators/dialogs/interpolation/interpolation.component.html
index f660fd7a..85d49b7b 100644
--- a/projects/core/src/lib/operators/dialogs/interpolation/interpolation.component.html
+++ b/projects/core/src/lib/operators/dialogs/interpolation/interpolation.component.html
@@ -71,7 +71,7 @@
                 </mat-hint>
             </geoengine-operator-output-name>
 
-            <geoengine-symbology-creator #symbologyCreator formControlName="symbologyCreation"></geoengine-symbology-creator>
+            <geoengine-symbology-creator #symbologyCreator></geoengine-symbology-creator>
         </ng-container>
 
         <ng-container actions>
diff --git a/projects/core/src/lib/operators/dialogs/neighborhood-aggregate/neighborhood-aggregate.component.html b/projects/core/src/lib/operators/dialogs/neighborhood-aggregate/neighborhood-aggregate.component.html
index 554d3193..5a9e8fa4 100644
--- a/projects/core/src/lib/operators/dialogs/neighborhood-aggregate/neighborhood-aggregate.component.html
+++ b/projects/core/src/lib/operators/dialogs/neighborhood-aggregate/neighborhood-aggregate.component.html
@@ -99,7 +99,7 @@
                 </mat-error>
             </geoengine-operator-output-name>
 
-            <geoengine-symbology-creator #symbologyCreator formControlName="symbologyCreation"></geoengine-symbology-creator>
+            <geoengine-symbology-creator #symbologyCreator></geoengine-symbology-creator>
 
             <p *ngIf="lastError$ | async as lastError" class="error">{{ lastError }}</p>
         </ng-container>
diff --git a/projects/core/src/lib/operators/dialogs/raster-scaling/raster-scaling.component.html b/projects/core/src/lib/operators/dialogs/raster-scaling/raster-scaling.component.html
index ad37c715..9dac1c6f 100644
--- a/projects/core/src/lib/operators/dialogs/raster-scaling/raster-scaling.component.html
+++ b/projects/core/src/lib/operators/dialogs/raster-scaling/raster-scaling.component.html
@@ -124,7 +124,7 @@
                 </mat-hint>
             </geoengine-operator-output-name>
 
-            <geoengine-symbology-creator #symbologyCreator formControlName="symbologyCreation"></geoengine-symbology-creator>
+            <geoengine-symbology-creator #symbologyCreator></geoengine-symbology-creator>
         </ng-container>
 
         <ng-container actions>
```

However, if you need to propagate the component's error, you need to add this to the form I guess.

Reply:
https://github.com/geo-engine/geoengine-ui/pull/546/commits/a1677705c75e9599ec4ccc6cdf16df6022e3f889